TheMovieDb/Model/AuthorReview.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  AuthorReview.swift.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 01/11/21.
+//
+
+import Foundation
+import SwiftUI
+
+struct AuthorReview: Decodable {
+    let name: String
+    let username: String
+    var avatarPath: String?
+    let rating: Float?
+    
+    enum CodingKeys: String, CodingKey {
+        case name
+        case username
+        case avatarPath = "avatar_path"
+        case rating
+    }
+    
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        self.name = try container.decode(String.self, forKey: .name)
+        self.username = try container.decode(String.self, forKey: .username)
+        self.avatarPath = try? container.decode(String.self, forKey: .avatarPath)
+        self.rating = try? container.decode(Float.self, forKey: .rating)
+    }
```

Comment:
seems you are not changing anything different from API or manipulating data you can remove it :)

Reply:
I removed :)
```suggestion
    istruct Review: Decodable {
    let id: String
    let author: String
    let content: String
    let createdAt: String
    let updatedAt: String
    let url: String
    let authorDetails: AuthorReview
}

```